gbus/worker.go

```diff
@@ -321,6 +320,9 @@ func (worker *worker) processMessage(delivery amqp.Delivery, isRPCreply bool) {
 	if err == nil {
 		_ = worker.ack(delivery)
 	} else {
+		for _, handler := range handlers {
+			metrics.ReportHandlerExceededMaxRetries(handler.Name(), worker.log())
```

Comment:
think this metric should be replaced with a ReportMessageRejected metric that is called only once if the message is rejected and not per handler

Reply:
And the report should not include the handler name but the message name?

Reply:
I think maybe we should have both, cause it's on the line between the messages domain and the handlers

Reply:
@danielwitz  what benefit do we gain with reporting this metric on the handler level...won't it just create info overload that won't be used ?

Reply:
@rhinof thought about it some more and got to the conclusion that a metric about rejected message is exactly what we wanted. 
I'll add that metric instead of the `ExceededMaxRetries`